README.md

```diff
@@ -1,22 +1,56 @@
-# DBOS Transact Python
+## 🚀 DBOS Transact - Ultra-Lightweight Durable Execution in Python 🚀
+
+---
+
+📚 **Documentation**: Under Construction 🚧
+
+💬 **Join the Discussion**: [Discord Community](https://discord.gg/fMwQjeW5zg)
+
+---
+
 
 **DBOS Python is under construction! 🚧🚧🚧 Check back regularly for updates, release coming in mid-September!**
 
-DBOS Transact is a **Python library** for building durable and scalable applications.
+DBOS Transact is a **Python library** providing ultra-lightweight durable execution.
+For example:
+
+```python
```

Comment:
Let's add a real runnable example in README. It'll be nice if people can just try it out.